apps/nextjs/.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+.yarn/install-state.gz
```

Comment:
?

Reply:
It's in the latest create-next-app basic template 😅